benchmark/fio_benchmark.yml

```diff
@@ -1,4 +1,13 @@
 ---
+- name: Run pre-check
+  hosts:
+    - localhost
```

Comment:
Can we run this task as a local_action?  We recently were asked to remove localhost from the rpc-ceph playbooks